### PR TITLE
New package: python3-plumbum-1.6.9

### PR DESCRIPTION
**Pull request by juantascon**

### CODE REVIEW

**Comment by juantascon** [comment]
@travankor could you please review this pullrq

File: srcpkgs/python3-plumbum/template [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'python3-plumbum'
+pkgname=python3-plumbum
+_pkgname=plumbum
```
Comment (travankor) [inline srcpkgs/python3-plumbum/template:3]: get rid of this

File: srcpkgs/python3-plumbum/template [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+version=1.6.9
+revision=1
+archs=noarch
+wrksrc="${_pkgname}-${version}"
```
Comment (travankor) [inline srcpkgs/python3-plumbum/template:7]: use ${pkgname#*-} instead

File: srcpkgs/python3-plumbum/template [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+depends="python3"
+short_desc="Shell combinators"
+maintainer="Juan Tascon <juan@horlux.org>"
+license="MIT"
+homepage="https://plumbum.readthedocs.io"
+distfiles="${PYPI_SITE}/p/${_pkgname}/${_pkgname}-${version}.tar.gz"
```
Comment (travankor) [inline srcpkgs/python3-plumbum/template:15]: same as above

File: srcpkgs/python3-plumbum/template [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+pkgname=python3-plumbum
+_pkgname=plumbum
+version=1.6.9
+revision=1
+archs=noarch
+wrksrc="${_pkgname}-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+depends="python3"
+short_desc="Shell combinators"
```
Comment (travankor) [inline srcpkgs/python3-plumbum/template:11]: Library for writing shell script-like programs in Python 3

**Comment by juantascon** [comment]
@travankor comments addressed, please review again

File: srcpkgs/python3-plumbum/template [hunk 5]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'python3-plumbum'
+pkgname=python3-plumbum
+version=1.6.9
+revision=1
+archs=noarch
```
Comment (ericonr) [inline srcpkgs/python3-plumbum/template:5]: We no longer use noarch.

**Comment by abenson** [comment]
Seemingly abandoned.

**Pull request closed without merging**